### ISSUE TITLE
Send PR to Bisq to add Hush

### ISSUE THREAD

**Issue by leto** [MyHush/hush3#22]
https://docs.bisq.network/exchange/howto/list-asset.html

500 HUSH bounty for somebody to write the necessary code and send PR

**Comment by csharpee** [comment]
There are no address validation examples for a KMD smartchain via Bisq. I used Zcash as a model, although I do not think this will work since we do not use the` "t" `at the beginning of Hush3 addrs. 

For this reason, I begun reading KMD / Hush3 code to figure out how the address is created and validated. Would you or others know the hierarchy of how the process flows? 

As an example: `./src/hush-cli getnewaddress`

The command invokes hush-cli file, then that file invokes komodo-cli to retrieve a new address...please correct me if I am wrong. 

On opening of komodo-cli, the file encoding is not supported, reason I am asking the question here.

In conclusion, the only thing we need to verify is these lines of Java code:

```
    public Hush() {
        super("Hush", "HUSH", new RegexAddressValidator("^t.*", "validation.altcoin.zAddressesNotSupported"));
    }

}
```

**Comment by leto** [comment]
@csharpee unless Bisq can support zaddrs, working on this is not useful, as we will be removing taddr support soon.

**Comment by csharpee** [comment]
Okay, they do not support z-addrs.
